alert-engine/src/test/scala/io/phdata/pulse/alertengine/trigger/AbstractAlertTriggerTest.scala

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Revert to old name?

Reply:
This test and the class being tested are new.